cmd/nope/main.go

```diff
@@ -1,7 +1,32 @@
 package main
 
-import "fmt"
+import (
+	"bufio"
+	"fmt"
+	"os"
+)
+
+// Don't know how to make this a constant
+var HISTFILE = os.Getenv("HOME") + "/.bash_history"
+
+func getLastLine(fname string) string {
+	file, err := os.Open(fname)
+
+	if err != nil {
+		fmt.Println("Can't open bash history file!")
+		panic(err)
+	}
+	defer file.Close()
+
+	scanner := bufio.NewScanner(file)
+
+	var x = ""
+	for scanner.Scan() {
```

Comment:
Not efficient, let's try to something like: http://stackoverflow.com/questions/10135738/reading-log-files-as-theyre-updated-in-go